src/app/shared/dynamic-form/dynamic-form.component.ts

```diff
@@ -109,7 +112,11 @@ export class DynamicFormComponent implements OnInit, OnDestroy {
   }
 
   updateFormRawValueWithObjects(): any {
-    const rawValueTmp: any = {};
+    const rawValueTmp: any = {
+      assigneeUser: {
+        uuid: 'S3ROT2-XAXJYF-VMIN7W-NA5ASJ7U',
```

Comment:
is this necessary? the call should work without it, no?

Reply:
removed. It was hardcoded only for the demo. My bad